pkg/rpcclient/doc.go

```diff
@@ -96,6 +96,8 @@ Supported methods
 Extensions:
 
 	getblocksysfee
+	getrawnotarypool
+	getrawnotarytransaction
```

Comment:
Adjust the commit message, please. The title can be something like "rpcclient: support `getrawnotarytransaction` and `getrawnotarypool` RPC methods"

> This function implements the RPC server's `getrawnotarytransaction` method

It doesn't implement. It *invokes*.

Reply:
Fixed.

Reply:
Not yet, there's still

> This function implements the RPC server's `getrawnotarypool` method.

Reply:
Yes, sorry. Fixed.